avro/message-status.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "messageStatus",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'message-status'",
+  "fields": [
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "name": "CrudOperation",
+        "symbols": [
+          "CREATE",
+          "UPDATE",
+          "REMOVE"
```

Comment:
Maybe is better "DELETE" (cruD)

Reply:
I used the same enum as service model https://github.com/pagopa/io-functions-elt/blob/55f04157aa3b157f96890eca627ba036b99c2e5e/avro/services.json#L15

Reply:
I appreciate the trust, but IMHO _REMOVE_ it's not a CRUD operations. 
We could use DELETE now and fix the service in the future.